bin/region_mover.rb

```diff
@@ -493,11 +493,30 @@ def getFilename(options, targetServer, port)
   return filename
 end
 
+def getServersRSGroup(servers, rsgroup_admin, hostname, port)
+  rsgroup = rsgroup_admin.getRSGroupOfServer(Address.fromParts(hostname,
+                                             java.lang.Integer.parseInt(port)))
+  return rsgroup unless rsgroup.nil?
+  begin
+    server_name = getServerName(servers, hostname, port)
+    regions = getRegions(getConfiguration(), server_name)
+    if !regions.nil? && regions.empty?
```

Comment:
Can regions be nil here by any chance? Might have to evaluate this to confirm:
`ProtobufUtil::getOnlineRegions(connection.getAdmin(ServerName.valueOf(servername)))`

Reply:
Checked the code. Currently, RSRpcService will return an empty list of regions as the RPC response or throw a RemoteServiceException.  It should be not nil at the client side. However, I also found that the ProtobufUtil used at client side will return null when the response is null. In my opinion, it is a guarantee here. Could be left as it is currently. What do you think?

Reply:
Sounds good. Thanks